src/sentry/runner/commands/backup.py

```diff
@@ -9,14 +9,18 @@
 from sentry.runner.decorators import configuration
 
 
+EXCLUDED_APPS = frozenset(("auth", "contenttypes"))
```

Comment:
Is the intent to not import users from one instance to another?

Reply:
This is the `auth` app not the model, which defines some pre-defined Django stuff as far as I can tell.

Reply:
My mistake. I remembered that our table is `auth_user` and thought it would be treated as from the 'auth' app, but I see now that our user class has switched the `app_label` as well.